src/main/java/racingcar/model/Winner.java

```diff
@@ -8,27 +8,24 @@
     private List<Car> cars;
 
     public Winner(List<Car> car) {
-        this.cars = car;
+        this.cars = new ArrayList<>(car);
     }
 
-    public List<String> getWinner() {
+    public List<String> findWinner() {
+        int maxPosition = findMaxPosition();
         List<String> winners = new ArrayList<>();
-        int maxPosition = getMaxPosition();
         for (Car car : cars) {
-            if (car.getPosition() == maxPosition) {
+            if (car.findPositionWithMax(maxPosition)) {
```

Comment:
아래와 같이 변경하는 것이 더 적절한 네이밍인 것 같아요. 🙂 
```suggestion
            if (car.isSamePosition(maxPosition)) {
```